licences-specs/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
 
 task mockTest(type: Test) {
   systemProperty 'geb.env', 'chromeHeadless'
+  systemProperty 'webdriver.chrome.driver', '/usr/local/bin/chromedriver/chromedriver'
```

Comment:
this could cause an issue when running locally but seems like webdriver isn't added to the path anymore in ci.